tt_metal/hw/inc/blackhole/noc_nonblocking_api.h

```diff
@@ -14,6 +14,10 @@
 #define NOC_0_X(noc_index, noc_size_x, x) (noc_index == 0 ? (x) : (noc_size_x - 1 - (x)))
 #define NOC_0_Y(noc_index, noc_size_y, y) (noc_index == 0 ? (y) : (noc_size_y - 1 - (y)))
 
+#define NOC_0_X_PHYS(noc_index, noc_size_x, x) NOC_0_X(noc_index, noc_size_x, x)
```

Comment:
I think you were following other conventions but can this and below just be `NOC_X/Y_PHYS` since it accepts the `noc_index`

Reply:
I was just following the convention used in the original NOC Encoding APIs. We have `NOC_0_X/Y` accepting `noc_index` as an arg, and `NOC_X/Y` just accepting `x/y` coords (these are defined in `risc_common.h` and use the global `noc_index` value).

I agree, these low level APIs should not be called `NOC_0*`, but I didn't want to change existing APIs as part of this effort. Does it make sense to you if we convert these to `NOC_X_COORDINATE/NOC_Y_COORDINATE` in a followup PR?

Reply:
works for me though maybe _COORD